File: WcfService/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WcfService
{
    public class Employee
    {
        string empId;
        string empFullname;
        string smsCode;
        string oldPassword;
        string newPassword;
        string action;

        public string EmpId
        {
            get
            {
                return empId;
            }

            set
            {
                empId = value;
            }
        }

        public string EmpFullname
        {
            get
            {
                return empFullname;
            }

            set
            {
                empFullname = value;
            }
        }

        public string SmsCode
        {
            get
            {
                return smsCode;
            }

            set
            {
                smsCode = value;
            }
        }

        public string OldPassword
        {
            get
            {
                return oldPassword;
            }

            set
            {
                oldPassword = value;
            }
        }

        public string NewPassword
        {
            get
            {
                return newPassword;
            }

            set
            {
                newPassword = value;
            }
        }

        public string Action
        {
            get
            {
                return action;
            }

            set
            {
                action = value;
            }
        }
    }
}

File: ui/src/main/java/com/sap/it/pmt/entity/Employee.java
package com.sap.it.pmt.entity;

import java.io.Serializable;

public class Employee implements Serializable {
	private static final long serialVersionUID = 1081545087359714262L;

	private String empId;
	private String empFullname;
	private String mobile;
	private String smsCode;
	private String oldPassword;
	private String newPassword;
	private String action;
	
    public String getEmpId() {
		return empId;
	}
	public void setEmpId(String empId) {
		this.empId = empId;
	}
	public String getEmpFullname() {
		return empFullname;
	}
	public void setEmpFullname(String empFullname) {
		this.empFullname = empFullname;
	}
	public String getMobile() {
        return mobile;
    }
    public void setMobile(String mobile) {
        this.mobile = mobile;
    }
    public String getSmsCode() {
		return smsCode;
	}
	public void setSmsCode(String smsCode) {
		this.smsCode = smsCode;
	}
	public String getOldPassword() {
		return oldPassword;
	}
	public void setOldPassword(String oldPassword) {
		this.oldPassword = oldPassword;
	}
	public String getNewPassword() {
		return newPassword;
	}
	public void setNewPassword(String newPassword) {
		this.newPassword = newPassword;
	}
	public String getAction() {
		return action;
	}
	public void setAction(String action) {
		this.action = action;
	}
    
    
}
